frontend/src/modules/activity/components/activity-platform-tabs.vue

```diff
@@ -97,6 +113,15 @@ export default {
     &--active {
```

Comment:
If we're not using this class anymore, let's just remove it 👍

Reply:
It is still being used by the `All Platforms` button.